### ISSUE TITLE
python issue with 3.6 and 3.7

### ISSUE THREAD

**Issue by matthiasplum** [AndresTanasijczuk/SlurmDagman#1]
I installed the package with pip but I'm running into this python issue with 3.6 and 3.7
```
Error running slurm_dagman:
Traceback (most recent call last):
  File "/home/plum/.local/bin/slurm_dagman", line 33, in init
    SLURM_DAGMAN_WORKER = Worker(options['dag_file'], options['outfile'], options['proxy_file'], options['sleep_time'], options['max_jobs_queued'], options['max_jobs_submit'], options['submit_wait_time'])
  File "/home/plum/.local/lib/python3.7/site-packages/SlurmDagman/process/worker.py", line 52, in __init__
    self.set_params(sleep_time, max_jobs_queued, max_jobs_submit, submit_wait_time)
  File "/home/plum/.local/lib/python3.7/site-packages/SlurmDagman/process/worker.py", line 151, in set_params
    self.__set_process_config_params()
  File "/home/plum/.local/lib/python3.7/site-packages/SlurmDagman/process/worker.py", line 156, in __set_process_config_params
    self.process_config.set_param('DAGMAN', 'sleep_time', self.sleep_time)
  File "/home/plum/.local/lib/python3.7/site-packages/SlurmDagman/config/manager.py", line 53, in set_param
    self.config.set(section, option, value)
  File "/cvmfs/icecube.opensciencegrid.org/py3-v4.1.1/RHEL_7_x86_64/lib/python3.7/configparser.py", line 1197, in set
    self._validate_value_types(option=option, value=value)
  File "/cvmfs/icecube.opensciencegrid.org/py3-v4.1.1/RHEL_7_x86_64/lib/python3.7/configparser.py", line 1182, in _validate_value_types
    raise TypeError("option values must be strings")
TypeError: option values must be strings

```
and if I do a str-typecast hack to fix this, i run into this problem:
```
Error running slurm_dagman:
Traceback (most recent call last):
  File "/usr/local/bin/slurm_dagman", line 42, in run
    return SLURM_DAGMAN_WORKER.run()
  File "/usr/local/lib64/python3.6/site-packages/SlurmDagman/process/worker.py", line 579, in run
    rc = self.__execute_dag()
  File "/usr/local/lib64/python3.6/site-packages/SlurmDagman/process/worker.py", line 545, in __execute_dag
    self.__handle_process_config_loading_parsing_and_writing()
  File "/usr/local/lib64/python3.6/site-packages/SlurmDagman/process/worker.py", line 236, in __handle_process_config_loading_parsing_and_writing
    elif not self.__parse_process_config() or not self.__validate_process_config():
  File "/usr/local/lib64/python3.6/site-packages/SlurmDagman/process/worker.py", line 202, in __parse_process_config
    = params[:]
TypeError: 'odict_values' object is not subscriptable

```

**Comment by AndresTanasijczuk** [comment]
Thank you for reporting this issue. My fault to advertise that the package is compatible with python 3 when I only tested it with python 2.7.
I will try to make it compatible with python 3.

**Comment by AndresTanasijczuk** [comment]
I just made a new release v0.1.3 which should have the python3 issues fixed (tested on python 3.6).
